pbf/views.py

```diff
@@ -131,8 +130,6 @@ def change_scenario(request, game_id):
 spirit_presence = {
```

Comment:
Does this section also need to be updated for Exploratory Shadows' changed energy?

Reply:
Ah never mind, scrolled back and found the flat +1 in models.py